go/libkb/saltpack_dec.go

```diff
@@ -11,13 +11,19 @@ import (
 
 func SaltPackDecrypt(
 	source io.Reader, sink io.WriteCloser,
-	deviceEncryptionKey NaclDHKeyPair) error {
-	_, plainsource, frame, err := saltpack.NewDearmor62DecryptStream(
+	deviceEncryptionKey NaclDHKeyPair, checkSender func(*saltpack.MessageKeyInfo) error) error {
```

Comment:
This 80-char wrapping seems particularly unreadable...perhaps it looks ok on punch cards, but...